lib/run-status.js

```diff
@@ -126,6 +128,17 @@ RunStatus.prototype.handleTest = function (test) {
 	this.emit('test', test, this);
 };
 
+RunStatus.prototype.formatAssertionError = function (error) {
```

Comment:
This kinda feels like the wrong place to be doing this.

Maybe in the individual reporters?